src/components/side_panel/custom_currency/custom_currency.ts

```diff
@@ -193,3 +195,7 @@ export class CustomCurrencyPanel extends Component<any, SpreadsheetChildEnv> {
     return currency.name + (currency.code ? ` (${currency.code})` : "");
   }
 }
+
+CustomCurrencyPanel.props = {
+  onCloseSidePanel: Function,
```

Comment:
add it to the props. Actually, it might be nice to have some sort of props check in typescript for every component added to the sidePanelRegistry (i.e. all should have a onCloseSidepanel in its props) Don't know if it's easy to do, doesn't look like it

Reply:
maybe all inherit from a component w/ that props? really don't know

Reply:
Don't know, same goes for tooltips :/
For now I let as it without any good solution